orcid-web/src/main/webapp/static/javascript/script.js

```diff
@@ -131,42 +131,35 @@ var OrcidCookie = new function() {
 
 var OrcidGA = function() {
     // test and make sure _gaq is working. disconnect.me chrome plugin has
-    // caused silent _gaq failures. This check allows us to detect that
-    // situation
-    var gaEnabled = false;
-    if(window.ga && ga.create) {
-    	gaEnabled = true;
-    }
-    
     this.buildClientString = function(clientGroupName, clientName) {
         return clientGroupName + ' - ' + clientName
     };
     this.gaPush = function(trackArray) {
-        //Recheck ga is enabled
-        if(!gaEnabled) {
-            if(window.ga && ga.create) {
-                gaEnabled = true;
-            }
-        }
-        
-        if (gaEnabled) {
-        	if(typeof trackArray === 'function') {
-        		ga(trackArray);
-        	} else {
-        		if(trackArray[5] == undefined) {
-                	ga(trackArray[0], trackArray[1], trackArray[2], trackArray[3], trackArray[4]);
+        if(window.ga) {
```

Comment:
Hey @rcpeters,

Look at these changes: 

1) First we check if ga was loaded (if(window.ga)), usually on dev env we dont load the ga scripts, so window.ga is null; so, we need to check if GA was loaded before proceed
2) Fill the ga array according to the trackArray (line 140, 143 and 145)
3) On line 150 we wait 200 ms, then, we check for ga.create, if it is not defined it means that GA was not loaded, so, we call th trackArray to continue with the normal flow of the site
4) The else statement for 1, if window.ga was undefined, call the trackArray

Please let me know if you see any issues.

Thanks